src/cov.jl

```diff
@@ -131,6 +131,13 @@ deviations `s`. Use `StatsBase.cov2cor!` for an in-place version.
 """
 cov2cor(C::AbstractMatrix, s::AbstractArray) = cov2cor!(copy(C), s)
 
+"""
+     cov2cor(C)
+
+ Compute the correlation matrix from the covariance matrix `C`. Use `StatsBase.cov2cor!` for an in-place version.
+ """
```

Comment:
I'd merge this with the docstring for the 2-argument version (with `cov2cor(C[, s])` as a signature).